spec/rspec/core/bisect/example_minimizer_spec.rb

```diff
@@ -19,6 +19,81 @@ module RSpec::Core
       expect(minimizer.repro_command_for_currently_needed_ids).to eq("rspec ex_2 ex_4 ex_5")
     end
 
+    it 'reduces a failure where none of the passing examples are implicated' do
+      no_dependents_runner = FakeBisectRunner.new(
+        %w[ ex_1 ex_2 ],
+        %w[ ex_2 ],
+        {}
+      )
+      minimizer = Bisect::ExampleMinimizer.new(no_dependents_runner, RSpec::Core::NullReporter)
+      minimizer.find_minimal_repro
+      expect(minimizer.repro_command_for_currently_needed_ids).to eq("rspec ex_2")
+    end
+
+    it 'reduces a failure when more than 50% of examples are implicated' do
+      fake_runner.always_failures = []
+      fake_runner.dependent_failures = { "ex_8" => %w[ ex_1 ex_2 ex_3 ex_4 ex_5 ex_6 ] }
+      minimizer = Bisect::ExampleMinimizer.new(fake_runner, RSpec::Core::NullReporter)
+      minimizer.find_minimal_repro
+      expect(minimizer.repro_command_for_currently_needed_ids).to eq(
+        "rspec ex_1 ex_2 ex_3 ex_4 ex_5 ex_6 ex_8"
+      )
+    end
+
+    it 'reduces a multiply-dependent failure' do
```

Comment:
Do you mean "multiple-dependent failure" instead of "multiply-dependent failure"?